lib/libutee/include/remoteproc_pta.h

```diff
@@ -0,0 +1,166 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (C) 2023, STMicroelectronics
+ */
+
+#ifndef __REMOTEPROC_PTA_H
+#define __REMOTEPROC_PTA_H
+
+#include <stdint.h>
+#include <util.h>
+
+/*
+ * Interface to the pseudo TA which provides platform implementation
+ * of the remote processor management
+ */
+
+#define PTA_RPROC_UUID { 0x54af4a68, 0x19be, 0x40d7, \
+		{ 0xbb, 0xe6, 0x89, 0x50, 0x35, 0x0a, 0x87, 0x44 } }
+
+/* Firmware format */
+#define PTA_RPROC_PROPRIETARY_FMT	BIT32(0)
+#define PTA_RPROC_ELF_FMT		BIT32(1)
+
+/* Firmware image protection */
+/* The platform supports copy of the input firmware image in secure memory */
+#define PTA_RPROC_FW_SECURE_COPY	BIT32(0)
+/* The platform supports load of segment with hash protection */
+#define PTA_RPROC_FW_WITH_HASH_TABLE	BIT32(1)
+/* The platform is able to change access to secure the firmware input image */
```

Comment:
a bit unclear.
`The platform is can restrict access remoteproc firmware memories`?